Cognite.Simulator.Utils/ModelLibraryBase.cs

```diff
@@ -252,7 +253,7 @@ private async Task<T> GetOrAddModelRevisionImpl(string modelRevisionExternalId,
 
             if (!downloaded && modelState.DownloadAttempts < _config.MaxDownloadAttempts)
```

Comment:
side question: do we currently retry downloads?

Reply:
this is exactly what it does, yes